Formula/kubernetes-helm@2.12.2.rb

```diff
@@ -0,0 +1,54 @@
+class KubernetesHelm < Formula
```

Comment:
This is a newer version of helm than we have on the Kube server -- are we going to upgrade that?

Reply:
Balls, I misread my `brew info kubernetes-helm` and extracted the wrong one.  Good catch. I'll update

Reply:
Just uploaded the correct one. I think we'll want this one soon too, so leaving it here.